ts/components/session/ActionsPanel.tsx

```diff
@@ -208,6 +257,51 @@ export const ActionsPanel = () => {
     return () => global.clearTimeout(timeout);
   }, []);
 
+  const getOnionPathIndicator = () => {
+    const hasOnionPath = getOnionPathStatus();
+
+    // const update: OnionUpdate = {
+    //   nodes: [
+    //     {
+    //       ip: 'hi',
+    //       label: 'hi',
+    //       isConnected: Math.random() > 0.5,
+    //       isAttemptingConnect: Math.random() > 0.7,
+    //     },
+    //     {
+    //       ip: 'hi2',
+    //       label: 'hi2',
+    //       isConnected: Math.random() > 0.5,
+    //       isAttemptingConnect: Math.random() > 0.7,
+    //     },
+    //     {
+    //       ip: 'hi3',
+    //       label: 'hi3',
+    //       isConnected: Math.random() > 0.5,
+    //       isAttemptingConnect: Math.random() > 0.7,
+    //     },
+    //   ],
+    // };
+
+
+    // dispatch(updateOnionPaths(update));
+
+    // TEST: Stuff
+    //  let testNode: SnodePath = {
+    //     bad: false,
+    //     path: new Array<Snode>()
+    //   }
+
+    //   dispatch(updateOnionPaths(testNode));
+
+    // console.log('Is Onion Path found -', hasOnionPath);
+    setHasOnionPath(hasOnionPath);
+  };
+
+  useInterval(() => {
+    getOnionPathIndicator();
+  }, 1000);
```

Comment:
you should be able to get rid of that once you have the dispatch where it is needed